components/compositing/compositor.rs

```diff
@@ -2220,7 +2227,8 @@ impl<Window: WindowMethods> IOCompositor<Window> {
     fn initialize_compositing(&mut self) {
         if self.webrender.is_none() {
             let show_debug_borders = opts::get().show_debug_borders;
-            self.context = Some(rendergl::RenderContext::new(self.native_display.clone(),
+            // We can unwrap() native_display because it's only None when using webrender.
+            self.context = Some(rendergl::RenderContext::new(self.native_display.unwrap().clone(),
```

Comment:
Please use `.expect("native_display should be Some when not using Webrender")` instead of `.unwrap()`